### DISCUSSION TITLE
Welcome to supernote-obsidian-plugin Discussions!

### DISCUSSION THREAD

**Discussion by philips** [philips/supernote-obsidian-plugin#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by znchong** [comment]
Thank you so much!!! You are the reason why I am planning to buy supernote! Do you also plan to allow plugin to extract the keywords / heading from supernote into obsidian? that will be great! I also see that you are planning to allow "link" in supernote to be imported into obsidian.. LOVE LOVE LOVE this plan!! Thank you so much!!!!
